.gitignore

```diff
@@ -16,3 +16,4 @@ indica/git_version
 *.png
 *.prof
 *.lprof
+.ropeproject/
```

Comment:
Was this a temporary change or is it still necessary?

Reply:
They're files generated by a refactoring tool which shouldn't be committed afaict so I don't think it's necessary but doesn't hurt either.